examples/example-gateway/config/test.yaml

```diff
@@ -67,6 +67,9 @@ tchannel.port: 0
 tchannel.processName: example-gateway
 tchannel.serviceName: example-gateway
 tchannel.clients.requestUUIDHeaderKey: x-request-uuid
+http.defaultHeaders:
+  Accept: application/json
+  Content-Type: application/json
```

Comment:
content in file is alphabetical. can you move this with the other http params, so it is easy for someone to search/edit as needed